Sources/SourceKit/sourcekitd/SwiftSourceKitFramework.swift

```diff
@@ -54,7 +58,11 @@ final class SwiftSourceKitFramework {
     // Workaround rdar://problem/43656704 by not constructing the value directly.
     // self.api = sourcekitd_functions_t(
     let ptr = UnsafeMutablePointer<sourcekitd_functions_t>.allocate(capacity: 1)
+    #if os(Windows)
+    memset(UnsafeMutableRawPointer(ptr), 0, MemoryLayout<sourcekitd_functions_t>.stride)
+    #else
     bzero(UnsafeMutableRawPointer(ptr), MemoryLayout<sourcekitd_functions_t>.stride)
```

Comment:
We should just use `memset` since it is cross-platform.

Reply:
Sure! Changed.